src/index.html

```diff
@@ -291,6 +287,17 @@ <h3>Keep Talking and Nobody Explodes</h3>
 					element.addEventListener("click", () => setTimeout(basicwires_solve));
 				});
 
+				// Complicated wires.
+				Array.from(document.querySelectorAll("#complicated_inputs .btn")).forEach((element) => {
+					element.addEventListener("click", buttonDeselect);
+					element.addEventListener("click", (e) => setTimeout(complicated_solve, 0, e));
+				});
+
+				Array.from(document.querySelectorAll("#complicated_optional .btn")).forEach((element) => {
+					element.addEventListener("click", buttonDeselect);
+					element.addEventListener("click", () => setTimeout(complicated_solve_many));
+				});
+
```

Comment:
Is this the result of a bad merge? It should have gone when complex wires was converted to typescript, right?
Same as the complex solve functions.